src/extended/FunctionNode.cpp

```diff
@@ -58,7 +57,7 @@ namespace lab
             _function(r, this, i, destP, n);
         }
 
-        _now += double(framesToProcess) / sampleRate();
+        _now += double(framesToProcess) / r.context()->sampleRate();
```

Comment:
seems like a case where the context should know now, and the function node might calculate _start - now()